subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/models/__init__.py

```diff
@@ -8,17 +8,16 @@
 
 class NotebookScaleMetadata(matbench_models.Metadata):
     test: enums.TestName
```

Comment:
what do you think about making this an `Union[enums.TestName, str]` to allow other values?